onadata/apps/messaging/serializers.py

```diff
@@ -99,3 +117,27 @@ def create(self, validated_data):
                         "Message not created. Please retry.")
                 else:
                     return instance
+
+
+@task
+def send_message(id, target_id, target_type, user, message_verb):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'id'](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5075103)